CHANGES.rst

```diff
@@ -11,6 +11,7 @@ Version master (UNRELEASED)
 - Adds progress report on workflow list response.
 - Makes CVMFS available in interactive sessions.
 - Adds preview flag to file download endpoint.
+- Creates REANA runtime components in the centrally configured (REANA-Commons) runtime namespace.
```

Comment:
I believe I added it here to not mix it with the user-facing changes, however, I see the next one is a fix that affects users as well, but it is a fix. We could come up with and order kind of:
1. User facing new features
2. User facing fixes
3. Architectural/Internal new features
4. Architectural/Internal fixes
5. Developer oriented new features
6. Developer oriented fixes

What do you think? cc @tiborsimko  

Whatever we decide, it would be great to document :)